### PR TITLE
Align files

### PR DESCRIPTION
**Pull request by architectbot**
This PR is generated by https://github.com/giantswarm/github

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,23 +4,31 @@
...
 - package-ecosystem: gomod
   directory: "/"
   schedule:
-    interval: daily
+    interval: weekly
     time: "04:00"
   open-pull-requests-limit: 10
   reviewers:
-  - giantswarm/team-firecracker-engineers
+  - giantswarm/team-firecracker
```
Comment (calvix) [inline .github/dependabot.yml:30]: i dont thnik we want this change

Comment (xh3b4sd) [reply .github/dependabot.yml:30]: /cc @MarcelMue @kopiczko

Comment (MarcelMue) [reply .github/dependabot.yml:30]: Comes from this value: https://github.com/giantswarm/github/blob/1773a9a7696c1783016c61a1f93cb3155d171b94/repositories/meta.yaml#L61

IMO could just be changed to the `-engineers` one

Comment (xh3b4sd) [reply .github/dependabot.yml:30]: See https://github.com/giantswarm/github/pull/42.

Comment (xh3b4sd) [inline .github/dependabot.yml:30]:
```suggestion
  - giantswarm/team-firecracker-engineers
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -4,10 +4,26 @@
...
+  reviewers:
+  - giantswarm/team-firecracker
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  target-branch: master
+  reviewers:
+  - giantswarm/team-firecracker
```
Comment (xh3b4sd) [inline .github/dependabot.yml:22]:
```suggestion
  - giantswarm/team-firecracker-engineers
```

**Review by xh3b4sd — APPROVED** [review]

**Pull request merged**